### DISCUSSION TITLE
What should I set the base tag's href attibute to?

### DISCUSSION THREAD

**Discussion by BarelySmooth** [theforumhelpers/theforumhelpers.github.io#246] [Q&A] [answered]
`<base href="https://theforumhelpers.github.io/index.html"> <!--If making local changes, change the base tag to point to how your filesystem is set up-->`

What should I set the ```href``` attribute to in the 404 page?

For the purposes of this topic, assume that the TFH website folder is located in ```C:/Users/abc123/Downloads/theforumhelpers.githib.io```

**Comment by Accio1** [comment, marked as answer]
In the case above, it should be set to "C:/Users/abc123/Downloads/theforumhelpers.githib.io/"

**Reply by BarelySmooth** [reply]
Thanks!

**Reply by BarelySmooth** [reply]
One more question: I'm on a Windows PC and it uses backslashes for paths, will this still work?

**Reply by Accio1** [reply]
I think so
